pkg/lakom/verifysignature/verifier.go

```diff
@@ -86,6 +87,12 @@ func verify(ctx context.Context, imageRef name.Reference, keys []crypto.PublicKe
 			}
 
 			if IsNoMatchingSignature(err) {
+				if strings.Contains(err.Error(), "context canceled") {
```

Comment:
Nit: I think that this is the preferred way to check for the context canceled error, but since this is some mitigation code both work for me.
```suggestion
				if errors.Is(err, context.Canceled) {
```